epub33/explainers/EPUB-33-security-privacy.md

```diff
@@ -0,0 +1,103 @@
+* [EPUB 3.3 Core](https://w3c.github.io/epub-specs/epub33/core/), the specification of the file format
+* [EPUB 3.3 Reading Systems](https://w3c.github.io/epub-specs/epub33/rs/), the specification for user agents, known as EPUB Reading Systems. 
+* [EPUB Accessibility 1.1](https://w3c.github.io/epub-specs/epub33/a11y/)
+* [EPUB Accessibility Techniques 1.1](https://w3c.github.io/epub-specs/epub33/a11y-tech/)
+
+
+
+## Introduction
+
+EPUB files are presented to end users by EPUB Reading Systems, which can be constructed as web applications but are more likely to be native apps. The EPUB specifications define the file format in great detail, but generally give user agents wide latitude, subject mostly to the requirements of the embedded technologies, such as HTML, SVG, XML, and CSS. 
+
+This makes it hard to think about security issues. Most of the security implications around EPUB depend on the architectural details and the business model of the reading system, rather than the specifications themselves. 
+
+It’s hard to even reason about how HTML’s security issues affect EPUB. If I read an EPUB on the Google Play app on an Android tablet, what risks am I exposed to? Can any of those risks be mitigated by the design of the EPUB specification?
+
+## Questionnaire
+
+### 1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+EPUB Reading Systems typically collect information on the reading habits of their users. In some cases this information is necessary to provide a good user experience; in other cases the information is collected for other users—the reading system company or the publisher of the book. 
+
+1. Tracking reading position. You really do want the reading system to remember that you just started chapter 5. 
+2. Remembering highlights, notes, bookmarks. 
+3. Remembering user preferences around fonts, visual themes (including dark/light), text justification, etc. 
+4. Recording progress through a book and whether a book was finished. 
+5. Some reading systems track every user reading session, including the time of day, the duration, how many pages were read, what book, the user’s IP address if a web-based reader, etc. 
+
+
+
+### 2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+The EPUB specification does not mention reading system features like bookmarks or highlights. There is no requirement to remember reading state. 
+
+### 3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+The EPUB specifications do not mention personal information.
+
+### 4. How does this specification deal with sensitive information?
+
+Merely the fact that you are reading a particular book can be sensitive information. Again, the specification offers no guidance to reading systems on this matter. 
+
```

Comment:
Worth noting that the EPUB content itself does not contain, per specification, any sensitive information.

Reply:
Added.